api/sendfild/main.go

```diff
@@ -186,14 +196,16 @@ var rootCmd = &cobra.Command{
 		cmd.ErrCheck(err)
 
 		conf := service.Config{
-			Listener:           listener,
-			ClientBuilder:      cb,
-			MongoUri:           mongoUri,
-			MongoDbName:        mongoDb,
-			MessageWaitTimeout: messageTimeout,
-			MessageConfidence:  messageConfidence,
-			RetryWaitFrequency: retryWaitFrequency,
-			Debug:              debug,
+			Listener:            listener,
+			ClientBuilder:       cb,
+			MongoUri:            mongoUri,
+			MongoDbName:         mongoDb,
+			MessageWaitTimeout:  messageTimeout,
+			MessageConfidence:   messageConfidence,
+			RetryWaitFrequency:  retryWaitFrequency,
+			AllowedFromAddrs:    allowedFromAddrs,
+			AllowEmptyFromAddrs: len(allowedFromAddrs) == 1 && allowedFromAddrs[0] == "*",
```

Comment:
Ok @textileben here it is. I view `*` as CLI sugar kind of thing, so that still needs to be translated into real data here to be passed into the `Service` constructor.